codestodians/komlie-frontend
# [Feature] Remove duplicated quantity field in cart

#### What does this PR do?
- Removes the duplicated quantity field in the cart

#### Description of Task to be completed?
- Fix bug with top right display on navbar (sometimes goes blank)
- Fix bug with non-clearing of cart items number on logout.
- Remove the extra input field for mutation of cart items number in cart.
- Prevent addition of more than items in stock of an item to cart on product-details page

#### How should this be manually tested?
- Check out into the `feature/remove-duplicated-quantity-cart-field` branch.
- Run the app
- Attempt adding or removing items from cart at your discretion.
- Also check for the duplicated field removed if it meets specification
